server/updaters.coffee

```diff
@@ -191,5 +191,12 @@ module.exports = ->
         incident.modifiedDate = new Date()
         Incidents.update(_id: incident._id, incident)
 
+  console.log "Removing incidents with invalid dates..."
+  invalidDateIncidents = []
+  Incidents.find().map (incident) ->
+    if incident.dateRange.end < incident.dateRange.start
+      invalidDateIncidents.push(incident._id)
+  console.log "Removing #{invalidDateIncidents.length} incidents"
+  Incidents.remove(_id: $in: invalidDateIncidents)
```

Comment:
Should we remove the entire incident? What if the incident's other data is accurate?